### ISSUE TITLE
Malignant hyperthermia w/ anesthesia

### ISSUE THREAD

**Issue by kanems** [obophenotype/human-phenotype-ontology#9728]
**Preferred term label:**
Malignant hyperthermia with anesthesia
**Synonyms**
Malignant hyperthermia with anaesthesia

**Definition (free text, please give PubMed ID)**
Malignant hyperthermia that is characterized by a rapid increase in temperature to 39-42 degrees C in response to inhalational anesthetics such as halothane or to muscle relaxants such as succinylcholine.

**Parent term (use hpo.jax.org/app)**
Malignant hyperthermia HP:0002047

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
From the parent term, I'm not sure which are anesthesia induced vs. exercise, this would need investigation..

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034733
